var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -134,9 +138,10 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
 
     # RAJA > 0.14 and Umpire > 6.0 require c++ std 14
     # We are working on supporting newer Umpire/RAJA versions
-    depends_on("raja@0.14.0:0.14", when="@0.5.0:+raja")
-    depends_on("umpire@6.0.0:6", when="@0.5.0:+raja")
-    depends_on("camp@0.2.3:0.2", when="@0.5.0:+raja")
+    depends_on("raja@0.14.0:0.14", when="@0.3.99:+raja")
+    depends_on("umpire@6.0.0:6", when="@0.3.99:+raja")
+    depends_on("camp@0.2.3:0.2", when="@0.3.99:+raja")
+    conflicts("+raja", when="@:0.3.0", msg="RAJA support requires HiOp > v0.3")
```

Comment:
I think we can just change the RAJA variant to be conditional instead of this conflict.

`variant("raja", default=False, when="@:0.3.0", description="Enable/Disable RAJA")`

Reply:
I suspect you meant `when="@0.3.99:"`. Added in 11df808814.

Reply:
Yes I did 🥲 thank you